tests/conftest.py

```diff
@@ -62,16 +64,17 @@ def mock_coordinator_service():
     server = grpc.server(futures.ThreadPoolExecutor(max_workers=1))
     agg = ModelSumAggregator()
     ctrl = IdController()
-    coordinator = Coordinator(
-        num_rounds=2,
-        minimum_participants_in_round=1,
-        fraction_of_participants=1.0,
-        aggregator=agg,
-        controller=ctrl,
-    )
-    with mock.patch("xain_fl.coordinator.coordinator_grpc.AbstractStore") as mock_obj:
+    with mock.patch("xain_fl.coordinator.store.AbstractStore") as mock_obj:
```

Comment:
perhaps for consistency this should be whatever mock store is used above, rather than calling upon the mocking framework (at the time I wasn't sure how "sufficiently mocked" `FakeS3Store` was so did it this way).

Reply:
I removed it. By default the `Coordinator` uses the `DummyStore` that does nothing, so we don't need a mock indeed.